codegen.yml

```diff
@@ -15,11 +15,24 @@ generates:
           content: export const possibleTypes = result.possibleTypes;
     config:
       apolloClientVersion: 3
-  ./src/api/graphql.generated.tsx:
+  ./src/api/graphql.generated.ts:
+    plugins:
+      - typescript
+    config:
+      reactApolloVersion: 3
+      scalars:
+        Date: string
+        DateTime: string
+      skipTypename: false
+      enumsAsTypes: true
+  ./src/:
+    preset: near-operation-file
+    presetConfig:
+      extension: .generated.tsx
```

Comment:
I don't think there's an JSX in these files, right? We could change it `.ts` and limit our eslint extensions.